### PR TITLE
Reverse Nix highlights queries

### PR DESCRIPTION
**Pull request by postsolar**
Related issue: #9436

I looked into injections too, ~~and most of them don't work for me (eg builders). But they don't work for me on master either, so I'm not sure what's the issue, and at least reversing doesn't seem to introduce any new breakage.~~ nevermind they seem to all work.

### CODE REVIEW

File: runtime/queries/nix/highlights.scm [hunk 1]
```diff
@@ -17,6 +17,8 @@
   "with" 
 ] @keyword
 
+(variable_expression name: (identifier) @variable)
```
Comment (the-mikedavis) [inline runtime/queries/nix/highlights.scm:20]: We also need to move these two patterns up in the file, right after this pattern :point_up::

```scm
(select_expression
  attrpath: (attrpath attr: (identifier)) @variable.other.member)

(apply_expression
  function: [
    (variable_expression name: (identifier) @function)
    (select_expression
      attrpath: (attrpath
        attr: (identifier) @function .))])
```

For example in the `flake.nix` in this repo it fixes the highlighting for `import nixpkgs` (the `apply_expression` pattern) and `builtin.path` (the `select_expression` papttern)

**Review by the-mikedavis — CHANGES_REQUESTED** [review]
It looks like there are a few more patterns to move but otherwise this looks like it's working :+1:

File: runtime/queries/nix/highlights.scm (hunk 1 above)
Comment (postsolar) [reply runtime/queries/nix/highlights.scm:20]: Fixed, thanks!

**Review by the-mikedavis — APPROVED** [review]
Thanks!

**Pull request merged**